wallet-new/server/Main.hs

```diff
@@ -173,6 +177,11 @@ actionWithWallet coreConfig txpConfig sscParams nodeParams ntpConfig params =
     plugins w dbMode = mconcat
         -- The actual wallet backend server.
         [ Plugins.apiServer pm params w
+            -- Enable CORS.
```

Comment:
Middlewares can be listed top level now the same way we list Plugins

Reply:
We can simple add here other middlewares from wai-extra for example http://hackage.haskell.org/package/wai-extra